### PR TITLE
<Tag> fix focus

### PR DESCRIPTION
**Pull request by AnastasiiaMashoshyna**

### CODE REVIEW

File: packages/wix-ui-core/src/components/tags-list/Tag.tsx [hunk 1]
```diff
@@ -87,60 +88,90 @@ export class Tag extends React.Component<TagProps> {
     this.inputRef.current.click();
   };
 
+  renderLabledInput = (ownProps, wrapperProps = {}) => {
+      const {
```
Comment (zonzujiro) [inline packages/wix-ui-core/src/components/tags-list/Tag.tsx:92]: Looks like you have missed prettier here.

File: packages/wix-ui-core/src/components/tags-list/Tag.tsx [hunk 2]
```diff
@@ -87,60 +88,90 @@ export class Tag extends React.Component<TagProps> {
     this.inputRef.current.click();
   };
 
+  renderLabledInput = (ownProps, wrapperProps = {}) => {
```
Comment (zonzujiro) [inline packages/wix-ui-core/src/components/tags-list/Tag.tsx:91]:
```suggestion
  renderLabeledInput = (ownProps, wrapperProps = {}) => {
```

File: packages/wix-ui-core/src/components/tags-list/Tag.tsx [hunk 3]
```diff
@@ -162,3 +330,13 @@ export class Tag extends React.Component<TagProps> {
     );
   }
 }
+export interface TagExperimentWrapperProps extends TagProps {
+  isFocusExperimentOpen?: boolean
+}
+export const WrappedWithExperimentTag: React.FC<TagExperimentWrapperProps> = (props) => {
```
Comment (argshook) [inline packages/wix-ui-core/src/components/tags-list/Tag.tsx:336]: What is this for? It shouldn't be part of wix-ui-core imo

File: packages/wix-ui-core/src/components/tags-list/Tag.tsx [hunk 4]
```diff
@@ -22,9 +22,177 @@ export interface TagProps {
...
+    const tabIndex = isFocusDisabled ? -1 : 0;
 
-export class Tag extends React.Component<TagProps> {
+    const rootElementProps = {
+      onKeyDown: this.handleKeyDown,
+      className: classNames(style.tag, className),
+      ...wrapperStyles
+    };
+
+    const labledInputProps = {
```
Comment (argshook) [inline packages/wix-ui-core/src/components/tags-list/Tag.tsx:164]:
```suggestion
    const labeledInputProps = {
```

File: packages/wix-ui-core/src/components/tags-list/Tag.tsx [hunk 5]
```diff
@@ -22,9 +22,177 @@ export interface TagProps {
...
+        role="checkbox"
+        aria-checked={checked}
+        aria-disabled={disabled}
+        href={link}
+        rel={rel}
+        tabIndex={disabled ? -1 : 0}
+        onClick={this.handleTagClick}
+        {...rootElementProps}
+      >
+        {this.renderLabeledInput(labledInputProps)}
```
Comment (argshook) [inline packages/wix-ui-core/src/components/tags-list/Tag.tsx:188]:
```suggestion
        {this.renderLabeledInput(labeledInputProps)}
```

File: packages/wix-ui-core/src/components/tags-list/Tag.tsx [hunk 6]
```diff
@@ -22,9 +22,177 @@ export interface TagProps {
...
+      </a>
+    ) :  this.renderLabeledInput(labledInputProps, rootElementProps);
```
Comment (argshook) [inline packages/wix-ui-core/src/components/tags-list/Tag.tsx:190]:
```suggestion
    ) :  this.renderLabeledInput(labeledInputProps, rootElementProps);
```

File: packages/wix-ui-core/src/components/tags-list/Tag.tsx [hunk 7]
```diff
@@ -22,9 +22,177 @@ export interface TagProps {
   link?: string;
   rel?: string;
   compId?: string;
+  style?: Record<string, string>;
```
Comment (piratus) [inline packages/wix-ui-core/src/components/tags-list/Tag.tsx:25]:
```suggestion
  style?: React.CSSProperties;
```

File: packages/wix-ui-core/src/components/tags-list/Tag.tsx (hunk 3 above)
Comment (zonzujiro) [reply packages/wix-ui-core/src/components/tags-list/Tag.tsx:336]: We need to check the fix under the experiment. It is a temporary code and will be removed after verification of the fix.

Comment (piratus) [reply packages/wix-ui-core/src/components/tags-list/Tag.tsx]: @argshook updated the naming and moved experiment condition to `wix-ui-santa`

Comment (argshook) [reply packages/wix-ui-core/src/components/tags-list/Tag.tsx]: @piratus thanks, much better this way!

**Pull request merged**